tests/Builder/SourceFileRepositoryTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\Builder;
+
+use Doctrine\Website\Builder\SourceFile;
+use Doctrine\Website\Builder\SourceFileRepository;
+use PHPUnit\Framework\TestCase;
+use function usort;
+
+class SourceFileRepositoryTest extends TestCase
```

Comment:
[Scrutinizer](https://scrutinizer-ci.com/g/doctrine/doctrine-website/inspections/85686785-d93a-4eab-8364-25a203c01ff7/code-structure/class/Doctrine%5CWebsite%5CBuilder%5CSourceFileRepository?expandCoverage=1) is showing uncovered code. We need to cover that code or remove it, in case it is unreachable.